packages/server/user-admin-api/.env

```diff
@@ -7,4 +17,10 @@ CUSTOMER_OS_API: http://localhost:10000/admin/query
 CUSTOMER_OS_API_KEY: 13604691-6850-4f08-a568-d995fb57b4f3
 
 USER_ADMIN_API_KEY=cad7ccb6-d8ff-4bae-a048-a42db33a217e
+GOOGLE_OAUTH_CLIENT_ID=665454598503-g4qrcc9e4jk6ujbbg7s60mdfas880sfi.apps.googleusercontent.com
```

Comment:
I know this is a separate app than the one used in production,

but it is just as powerful, so I think these fields should be kept blank in git, kept in keypass, and document in a README to get these values from there